scripts/run-shellcheck.sh

```diff
@@ -2,8 +2,8 @@
 export LC_ALL="C"
 
 find "$@" -type f -print0 \
-    | xargs -0 sh -c 'for i in "$@"; do { printf "%s\n" "$i" | grep "\\.sh\$" \
-        || head -n1 "$i" | grep --text -Ev "(atf-|tcl|wi)sh\$" | grep --text "^#!.*sh\$"
+    | xargs -0 sh -c 'for i in "$@"; do { printf "%s\n" "$i" | grep -E "^.*\\.(ash|bash|bats|dash|ksh|sh)$" \
```

Comment:
@kdudka Isn't there missing `\`?

```suggestion
    | xargs -0 sh -c 'for i in "$@"; do { printf "%s\\n" "$i" | grep -E "^.*\\.(ash|bash|bats|dash|ksh|sh)$" \
```

Reply:
As discussed in person, both the variants seem to work equally but I will add the backslash for clarity.